packages/ui/src/testHelpers.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
```

Comment:
🤔 Why do we need this?

Reply:
This one comes from the other PR that implemented `withExpectedError`